frontend/mulighetsrommet-veileder-flate/src/api/api.ts

```diff
@@ -1,11 +1,11 @@
 import { ALL_TOGGLES, Features } from './features';
 import useFetch from '@nutgaard/use-fetch';
-import { APP_NAME } from '../utils/constants';
+import { APPLICATION_NAME } from '../constants';
 
 const headers = {
-  headers: { 'Nav-Consumer-Id': APP_NAME },
+  headers: { 'Nav-Consumer-Id': APPLICATION_NAME },
```

Comment:
Denne har ikke jeg lagt merke til før!

Det finnes noe lignende kode i `index.tsx`, så her kan vi kanskje gjøre noe refaktorering for å samle trådene :thinking:

Reply:
Refaktorerte nå! La den til i `utils.ts` istedet